pkg/server/chaosd/disk.go

```diff
@@ -124,17 +125,15 @@ func (diskAttack) attackDiskFill(fill *core.DiskCommand) error {
 			log.Error(fmt.Sprintf(" unexcepted err when parsing disk percent '%s'", fill.Percent), zap.Error(err))
 			return err
 		}
-		dir, err := os.Getwd()
+		dir := filepath.Dir(fill.Path)
+		s := syscall.Statfs_t{}
```

Comment:
why use this instead of `disk.GetInfo`?

Reply:
This is just the implements of disk.GetInfo. I change it mostly because minio is hard to import into go mod , and pass to ci.And minio is also not recommand to be used as a lib.